core/src/banking_stage/packet_receiver.rs

```diff
@@ -20,10 +24,14 @@ pub struct PacketReceiver {
 }
 
 impl PacketReceiver {
-    pub fn new(id: u32, banking_packet_receiver: BankingPacketReceiver) -> Self {
+    pub fn new(
+        id: u32,
+        banking_packet_receiver: BankingPacketReceiver,
+        bank_forks: Arc<RwLock<BankForks>>,
+    ) -> Self {
         Self {
             id,
-            packet_deserializer: PacketDeserializer::new(banking_packet_receiver),
+            packet_deserializer: PacketDeserializer::new(banking_packet_receiver, bank_forks),
```

Comment:
`bank_fork` is actually passed to `packet_deserializer`, which covers both non-vote and vote packets